ee/packet2/src/erl-support.c

```diff
@@ -0,0 +1,18 @@
+/*
+# _____     ___ ____     ___ ____
+#  ____|   |    ____|   |        | |____|
+# |     ___|   |____ ___|    ____| |    \    PS2DEV Open Source Project.
+#-----------------------------------------------------------------------
+# Copyright 2001-2004, ps2dev - http://www.ps2dev.org
+# Licenced under Academic Free License version 2.0
+# Review ps2sdk README & LICENSE files for further details.
+#
+# The erl-tags support
+*/
+
+#include <erl.h>
```

Comment:
I'm not sure if ERL still works, and I don't know of any examples using it. But as long as it's in ps2sdk we should try to keep it updated I guess.